### PR TITLE
Contributing and CLA

### LINKED ISSUE #205: Add Contributor Agreement to Appsody-Operator Repository
Need to add and implement a contributor agreement and check that all contributors have signed off.

See parent epic for more information: https://github.ibm.com/WASCloudPrivate/roadmap-cloudpaks/issues/679

### PR DESCRIPTION
**Pull request by JusteenR**
**What this PR does / why we need it?**:

- Adds contributing guidelines for users to follow 
- Adds the CLA assistant tool

**Which issue(s) this PR fixes**:

Fixes #205

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Appsody Application operator contributing guidelines
+Welcome to the Appsody community!
+
+You can contribute to the project in a variety of ways:
+
+* Fixing or reporting bugs
+* Improve documentation
+* Contribute code
+We welcome all contributions to the Appsody Application operator project and request you follow the guidelines below:
```
Comment (hsaunders0788) [inline CONTRIBUTING.md]: Should operator be capitalized in this sentence? Is it the Appsody Application Operator project?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+* Fork the affected repository.
+* Create a new branch for your changes.
+* Develop the code/fix.
+* Modify the documentation as necessary.
+* Verify all CI status checks pass, and work to make them pass if failing.
+The general rule is that all PRs should be 100% complete - meaning they should include documentation changes related to the change. A significant exception is work-in-progress PRs. These should be indicated by opening a draft pull request. To open a draft pull request, click the dropdown arrow that appears next to the “Create pull request” button and then select the "Create draft pull request" option.
+
+## Contributor License Agreement
+In order for us to merge any of your changes into Appsody Application operator, you need to sign the Contributor License Agreement. When you open up a Pull Request for the first time, a bot will comment with a link to the CLA. You can review or sign the CLA now here https://cla-assistant.io/appsody/appsody-operator
```
Comment (hsaunders0788) [inline CONTRIBUTING.md]: Same thing here, I think it either needs to read "the Appsody application operator" or "Appsody Application Operator"

File: readme.md [hunk 3]
```diff
@@ -24,3 +24,7 @@ The architecture of the Appsody Application Operator follows the basic controlle
 ## Documentation
 
 For information on how to use the `AppsodyApplication` operator, see the [documentation](doc/).
+
+## Contributing
+
+We welcome all contributions to the Appsody Application operator project. Please see our [Contributing guidelines](CONTRIBUTING.md)
```
Comment (hsaunders0788) [inline readme.md]: Ditto

File: CONTRIBUTING.md (hunk 1 above)
Comment (JusteenR) [reply CONTRIBUTING.md]: Changed the capitalization for all of the operators.

**Pull request merged**